Mathlib/Analysis/SpecificLimits/InvPow.lean

```diff
@@ -0,0 +1,79 @@
+/-
+Copyright (c) 2025 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import Mathlib.Analysis.Complex.TaylorSeries
+
+/-!
+# Taylor expansion of `1 / z ^ r`
+-/
+
+lemma deriv_div_sub_pow {𝕜 : Type*} [NontriviallyNormedField 𝕜] (z w a : 𝕜) (r : ℕ) :
+    deriv (fun z : 𝕜 ↦ a / (w - z) ^ r) z = r * a / (w - z) ^ (r + 1) := by
```

Comment:
I think it would be a good idea to state this with `HasDerivAt` so that you can get `Differentiable` for free as well.